eng/Versions.props

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <VersionPrefix>3.11.0</VersionPrefix>
     <PreReleaseVersionLabel>beta1</PreReleaseVersionLabel>
-    <NetAnalyzersVersionPrefix>8.0.0</NetAnalyzersVersionPrefix>
+    <NetAnalyzersVersionPrefix>9.0.0</NetAnalyzersVersionPrefix>
     <NetAnalyzersPreReleaseVersionLabel>preview</NetAnalyzersPreReleaseVersionLabel>
```

Comment:
There are two pre-release version labels @mmitche. One is "preview" as you suggested in the email, the other one is beta1, but I assume you'd like me to also set that to preview, correct?

Reply:
I wouldn't change the one that is set to `beta1`.